docs/news/unreleased/osbuild-rhsm-stage.md

```diff
@@ -4,3 +4,10 @@ Add support for `org.osbuild.rhsm` osbuild stage. This stage is available in
 osbuild since version 24. The stage currently allows only configuring the
 enablement status of two RHSM DNF plugins, specifically of `product-id` and
 `subscription-manager` DNF plugins.
+
+# RHEL 8.3 & 8.4: Disable all RHSM DNF plugins on qcow2 image
+
+Disable both available RHSM DNF plugins (`product-id` and
+`subscription-manager`) on rhel-8 and rhel-84 qcow2` images. The reason for
```

Comment:
There's an extra `.

Reply:
Nice catch... It took me a few seconds to spot it even now :smile: